features/testbot/bearing_param.feature

```diff
@@ -75,7 +75,10 @@ Feature: Bearing parameter
             #| 0    | b  | 189 189  | da          | 180                                   |
             | 0    | 1  | 90 270   | ab,bc,cd,cd    | 0->90,90->0,0->270,270->0           |
             | 1    | d  | 10 10    | bc,bc          | 0->0,0->0                           |
-            | 1    | d  | 90 90    | ab,bc,cd,da,da | 0->90,90->0,0->270,270->180,180->0  |
+            # The target point `a` can belong to either `ab` or `ha` edge.
+            # The latter case leads to "Impossible route between points",
+            # because `ab` and `ha` are in different components
+            #| 1    | d  | 90 90    | ab,bc,cd,da,da | 0->90,90->0,0->270,270->180,180->0 |
```

Comment:
The problem is in the target point for `d`. Here the nearest edges to `d`:
```
input_coordinate = (lon:1.000089, lat:1)
   2,1 "cd" distance 0.111226299998 fwd bearing 270.000007845 invalid, bck bearing 90.0000078449 invalid
   1,5 "da" distance 0.111226299998 fwd bearing 180 invalid, bck bearing 360 invalid
   1,0 "dg" distance 0.111226299998 fwd bearing 270.000000777 invalid, bck bearing 90.0000007767 invalid
   4,5 "ha" distance 10.1215933 fwd bearing 89.9999992235 valid, bck bearing 269.999999223 invalid
   5,6 "ab" distance 10.1215933 fwd bearing 90 valid, bck bearing 270 invalid 0.1112263
```
With the new ordering `ha` is before `ab` and `ha` also has bearing 90, so it is a valid phantom node, but `ha` is in another scc. One possibility would be to introduce point `2` close to `d` on the edge `cd` and use instead of `d`